VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.spec.ts

```diff
@@ -52,28 +43,24 @@ describe('ParticipantInfoTooltip component', () => {
     });
 
     it('strips out quick link prefix for quick link particpant hearing role', () => {
-        const expectedDisplayText = 'expectedDisplayText';
         component.participant = new ParticipantContactDetails(
             new ParticipantContactDetailsResponseVho({ role: Role.QuickLinkParticipant, id: 'id', hearing_role: 'Quick link participant' })
         );
-        translateService.instant.withArgs('participant-info-tooltip.quick-link-participant').and.returnValue(expectedDisplayText);
         fixture.detectChanges();
         const hearingRoleElement = fixture.debugElement.query(By.css('#tooltip-hearing-role-id'));
-        expect(hearingRoleElement.nativeElement.innerHTML).toContain(expectedDisplayText);
+        expect(hearingRoleElement.nativeElement.innerHTML).toContain(component.quickLinkParticipantDisplayText);
     });
 
     it('strips out quick link prefix for quick link observer hearing role', () => {
-        const expectedDisplayText = 'expectedDisplayText';
         component.participant = new ParticipantContactDetails(
             new ParticipantContactDetailsResponseVho({ role: Role.QuickLinkObserver, id: 'id', hearing_role: 'Quick link observer' })
```

Comment:
You could use the HearingRoles enum for this instead of a magic string.

Reply:
Actually following on from other comments is the hearing role not relevant for these at all?

Reply:
It is still relevant as this change is being made because the hearing role displays the prefix of quick link and we dont want it to be shown in the command center. I could have changed this in the participant model itself however that would change it everywhere hearing role is being used so I instead decided to make the change in this component to comply with the changes requested by Neil and Lorenzo